test/OpenTelemetry.Instrumentation.GrpcClient.Tests/GrpcClientTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿// <copyright file="GrpcClientTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
+using System.Diagnostics;
+using System.Linq;
+using Greet;
+using Grpc.Net.Client;
+using Moq;
+using OpenTelemetry.Instrumentation.GrpcClient.Tests.Services;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Trace.Configuration;
+using OpenTelemetry.Trace.Export;
+using Xunit;
+
+namespace OpenTelemetry.Instrumentation.GrpcClient.Tests
+{
+    public class GrpcClientTests : IClassFixture<GrpcFixture<GreeterService>>
+    {
+        private GrpcFixture<GreeterService> fixture;
+
+        public GrpcClientTests(GrpcFixture<GreeterService> fixture)
+        {
+            // Allows gRPC client to call insecure gRPC services
+            // https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-3.1#call-insecure-grpc-services-with-net-core-client
+            AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
```

Comment:
Do we need to turn this off after the test so it wont affect something else?

Reply:
No, it just effects https + HTTP/2. The switch stops it from automatically falling back to HTTP/1.1.